pkg/apis/infinispan/v1/types_util.go

```diff
@@ -369,3 +371,69 @@ func (ispn *Infinispan) StorageSize() string {
 	}
 	return ""
 }
+
+// AddLabelsForPods adds to the user maps the labels defined for pods in the infinispan CR
+func (ispn *Infinispan) AddLabelsForPods(uMap map[string]string) {
+	addLabelsFor(ispn, "infinispan.org/podTargetLabels", uMap)
+}
+
+// AddLabelsForServices adds to the user maps the labels defined for services and ingresses/routes in the infinispan CR
+func (ispn *Infinispan) AddLabelsForServices(uMap map[string]string) {
+	addLabelsFor(ispn, "infinispan.org/targetLabels", uMap)
+}
+
+// AddOperatorLabelsForPods adds to the user maps the labels defined for pods in the infinispan CR by the operator
+func (ispn *Infinispan) AddOperatorLabelsForPods(uMap map[string]string) {
+	addLabelsFor(ispn, "infinispan.org/operatorPodTargetLabels", uMap)
```

Comment:
Do we really need so complex name for annotation? It's our own CR and not a build-in and we can use it without `infinispan.org` prefix

Reply:
[docs](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set) say: `If the prefix is omitted, the annotation Key is presumed to be private to the user. `
imo it's ok to keep the prefix and avoid name clashing with names in user domain.